specification/ai/Azure.AI.Client/files/models.tsp

```diff
@@ -0,0 +1,129 @@
+import "@typespec/versioning";
```

Comment:
Can you please put everything related to Agents under the  
specification/ai/Azure.AI.Client/agents folder? You can have multiple subfolders there, like:
/agents/agents/
/agents/files/
/agents/messages/
...

Reply:
Why?

Reply:
I have put the files like this now to follow OpenAI assistants structure as close as possible